monai/networks/nets/swin_unetr.py

```
@@ -274,50 +328,51 @@ def __init__(
-                d.reduction.weight.copy_(wstate["module.layers4.0.downsample.reduction.weight"])  # type: ignore
-                d.norm.weight.copy_(wstate["module.layers4.0.downsample.norm.weight"])  # type: ignore
-                d.norm.bias.copy_(wstate["module.layers4.0.downsample.norm.bias"])  # type: ignore
+
+            for layer_name, stage in [
+                ("layers1", layers1_0),
+                ("layers2", layers2_0),
+                ("layers3", layers3_0),
+                ("layers4", layers4_0),
+            ]:
+                if _stage_is_hyena(stage):
+                    warnings.warn(
```

Ruff (0.15.18)

[warning] No explicit `stacklevel` keyword argument found

Set `stacklevel=2`

(B028)